flask_rest_jsonapi/resource.py

```diff
@@ -302,23 +302,14 @@ def get(self, *args, **kwargs):
         self.before_get(args, kwargs)
 
         relationship_field, model_relationship_field, related_type_, related_id_field = self._get_relationship_data()
-        related_view = self.schema._declared_fields[relationship_field].related_view
-        related_view_kwargs = self.schema._declared_fields[relationship_field].related_view_kwargs
 
         obj, data = self._data_layer.get_relationship(model_relationship_field,
                                                       related_type_,
                                                       related_id_field,
                                                       kwargs)
 
-        for key, value in copy(related_view_kwargs).items():
-            if isinstance(value, str) and value.startswith('<') and value.endswith('>'):
-                tmp_obj = obj
-                for field in value[1:-1].split('.'):
-                    tmp_obj = getattr(tmp_obj, field)
-                related_view_kwargs[key] = tmp_obj
```

Comment:
This line is the original source of the bug. 

It was rewriting the related_view_kwargs template with the value for the current page access. From this point on, related_view_kwargs will remain in a broken state until the application is restarted